mozetl/taar/taar_locale.py

```diff
@@ -37,6 +41,24 @@ def store_new_state(source_file_name, s3_dest_file_name, s3_prefix, bucket):
     transfer.upload_file(source_file_name, bucket, key_path)
 
 
+def load_amo_external_whitelist():
+    white_list_inner = []
+    # Load the most current AMO dump JSON resource.
+    try:
+        amo_dump = utils.get_s3_json_content_no_cache(AMO_DUMP_BUCKET, AMO_DUMP_KEY)
```

Comment:
Instead of importing all the stuff in the `utils.py` file, which is not really needed here, why don't you do:

```
s3 = boto3.client('s3')
s3_contents = s3.get_object(Bucket=AMO_DUMP_BUCKET, Key=AMO_DUMP_KEY)
amo_dump = json.loads(s3_contents['Body'].read())
```

Within the try?